### ISSUE TITLE
[feature request] Add support multicast addresses (FE02::1, FE02::2), specified in [RFC 4291, Section 2.7]

### ISSUE THREAD

**Issue by Bogdan107** [yggdrasil-network/yggdrasil-go#782]
Multicast Addresses specified in [RFC 4291 Section 2.7](https://datatracker.ietf.org/doc/html/rfc4291#section-2.7)

If I run ```ping FE02::1%wlan0```, I expect to receive ping responses from all addresses, that are configured on the interfaces of systems, connected to the same physical environment. In a real wireless network, using this command, I get ping responses from all systems on the wireless network (clients of the same access point that are connected to the same bridge on that access point).

yggdrasil is like a multi-port virtual switch where port 0 is always connected to the ygg0 host interface.
Therefore with the command ```ping FE02::1%ygg0``` I expect to receive ping responses from all peers, to which my yggdrasil service is connected to (all addresses, which configured on all ports of yggdrasil commutator - see first column of ```yggdrassil getpeers``` table).
But always I get ping responses only from local address, never from addresses of any connected peers.

Yes, I can find addresses of connected peers by parsing ```yggdrassil getpeers``` table, and then pinging them one-by-one, but it is not trivial, non-standart and non-usual way.

This feature is required by some services, that use multicast addresses, such as [A.L.F.R.E.D.](https://www.open-mesh.org/projects/alfred/wiki)

Multicast addresses are needed for sending broadcast packets to all connected systems in link-local broadcast domain. This packets not intended to be routed to third-party systems.

**Comment by tdemin** [comment]
You know Yggdrasil is supposed to be a tunnel, and not an L2 segment emulation, right?

**Comment by ShadowJonathan** [comment]
Sending broadcast over yggdrasil is troublesome for many reasons;
1. yggdrasil's network is not intended to be exhaustively mapped by one node, even though debug APIs exist today for that, they're eventually intended to be removed, you shouldn't be able to see through the "fog of war", beyond 1 or 2 node hops, exhaustively.
2. Even if all nodes are know, such broadcasts would create way much overhead, and generally be a lot more expensive than normal Ethernet or WLAN-based networks. There is also the potential for broadcast storms, and there is a *very good reason* why broadcasts packets are generally dropped immediately on the public internet.

In conclusion; Yggdrasil is not a LAN, it is a WAN.

**Comment by neilalexander** [comment]
Closing this as we’re not likely to action it. There are all sorts of problems involved in knowing how to route multicast and it isn’t really in scope for the project at this time.
